### ISSUE TITLE
Please tag a version 1.1.1

### ISSUE THREAD

**Issue by SvenRtbg** [guzzle/psr7#27]
I do need that feature of conditionally loading the global functions due to messed up autoloading on my side.

**Comment by mtdowling** [comment]
I'm waiting on https://github.com/guzzle/psr7/pull/25 for the next release.

**Comment by jhedstrom** [comment]
It would be nice to have 1.1.1 tagged so Drupal 8.0.0 can include the conditional function loading fix. See https://www.drupal.org/node/2546272

**Comment by mtdowling** [comment]
Still waiting on #25

**Comment by brantwynn** [comment]
Looks like #25 is ready for review now.
